src/test/java/bio/terra/service/dataset/SelfHostedDatasetIntegrationTest.java

```diff
@@ -92,18 +97,23 @@ public class SelfHostedDatasetIntegrationTest extends UsersBase {
   private List<String> uploadedFiles;
   private String ingestServiceAccount;
   private String ingestBucket;
+  private Map<String, String> directFileIngestTargetPathsToIds;
 
   @Before
   public void setup() throws Exception {
+    logger.info("Beginning test setup...");
     super.setup();
     stewardToken = authService.getDirectAccessAuthToken(steward().getEmail());
     dataRepoFixtures.resetConfig(steward());
     profileId = dataRepoFixtures.createBillingProfile(steward()).getId();
     uploadedFiles = new ArrayList<>();
+    directFileIngestTargetPathsToIds = new HashMap<>();
+    logger.info("Test setup complete.");
   }
 
   @After
   public void teardown() throws Exception {
+    logger.info("Beginning test teardown...");
```

Comment:
I left these log messages in on purpose: when tests fail prematurely, these made it easier to read through the logs and identify the turning point.